ui/src/message_popup/test_fixtures.js

```diff
@@ -0,0 +1,20 @@
+/* flow weak */
+import _ from 'lodash';
+import {allQuestions} from './questions.js';
+import {withStudents} from './transformations.jsx';
+
+
+/*
+These are fixtures for JavaScript tests
```

Comment:
The idea here is that these are only used in test, and they're split out here since we want to be able to re-use them in different test scenarios.

These will need to be updated as the code itself changes too.